analyses/models.py

```diff
@@ -195,12 +195,28 @@ def latest_analysis_status(self) -> dict["Analysis.AnalysisStates", bool]:
         latest_analysis: Analysis = self.latest_analysis
         return latest_analysis.status
 
-    def set_experiment_type_by_ena_library_strategy(self, ena_library_strategy: str):
-        if ena_library_strategy.lower() == "rna-seq":
+    def set_experiment_type_by_metadata(
+        self, ena_library_strategy: str, ena_library_source: str
+    ):
+        if (
+            ena_library_strategy.lower() == "rna-seq"
+            and ena_library_source.lower() == "metagenomic"
```

Comment:
Only making a note here in the hopes it helps us remember later when we work on it... but it would be cool if these (semi-) controlled vocab constants like `rna-seq` etc were part of the new ENA API Handler package.